packages/relic/test/headers/new/special/accept_encoding_header_test.dart

```diff
@@ -0,0 +1,331 @@
+import 'dart:io';
+import 'package:test/test.dart';
+import 'package:relic/src/headers/headers.dart';
+import 'package:relic/src/relic_server.dart';
+import '../../headers_test_utils.dart';
+
+void main() {
+  group('Given an Accept-Encoding header with the strict flag true', () {
```

Comment:
Suggestion/Discussion: When strict mode is active we probably only need to valide how the behavior changes. So if we already validate the details around parsing correct headers in a non-strict mode, then we can only validate when we should reject the header in the strict case?